kernelci/api/models.py

```diff
@@ -472,6 +486,14 @@ class RegressionData(BaseModel):
     pass_node: Optional[PyObjectId] = Field(
         description="Previous passing Node"
     )
+    node_sequence: Optional[List[PyObjectId]] = Field(
+        description=("Instances of this same job ran after the initial "
+                     "failure. The last run in the sequence may be a "
+                     "passed run, which means the regression is no longer"
+                     "active. If the sequence is empty or if all the runs "
+                     "in the sequence failed, that means the job is still "
+                     "failing and the regression is active")
+    )
```

Comment:
Related to https://github.com/kernelci/kernelci-pipeline/pull/503/files#r1544460596
```suggestion
    node_sequence: List[PyObjectId] = Field(
        default=[],
        description=("Instances of this same job ran after the initial "
                     "failure. The last run in the sequence may be a "
                     "passed run, which means the regression is no longer"
                     "active. If the sequence is empty or if all the runs "
                     "in the sequence failed, that means the job is still "
                     "failing and the regression is active")
    )
```